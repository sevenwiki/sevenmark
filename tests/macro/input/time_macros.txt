Current time: [now]
Age calculation: [age(1990-01-15)]
Line break:[br]Next line