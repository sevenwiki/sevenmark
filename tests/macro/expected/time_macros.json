[
  {
    "Text": {
      "location": {
        "start": 0,
        "end": 14
      },
      "content": "Current time: "
    }
  },
  "TimeNow",
  {
    "Text": {
      "location": {
        "start": 19,
        "end": 20
      },
      "content": "\n"
    }
  },
  {
    "Text": {
      "location": {
        "start": 20,
        "end": 37
      },
      "content": "Age calculation: "
    }
  },
  {
    "Age": {
      "location": {
        "start": 37,
        "end": 54
      },
      "content": "1990-01-15"
    }
  },
  {
    "Text": {
      "location": {
        "start": 54,
        "end": 55
      },
      "content": "\n"
    }
  },
  {
    "Text": {
      "location": {
        "start": 55,
        "end": 66
      },
      "content": "Line break:"
    }
  },
  "NewLine",
  {
    "Text": {
      "location": {
        "start": 70,
        "end": 79
      },
      "content": "Next line"
    }
  }
]